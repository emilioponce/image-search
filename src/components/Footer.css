.Footer {
  background-color: #202329;
  color: #ffffff;
  padding: 10px;
  font-size: 80%;
  position: fixed;
  left: 0px;
  bottom: 0px;
  width: 100%;
}

.Element {
  margin-left: 5px;
  margin-right: 5px;
}

.Element > a,
.Element-right > a {
  color: white;
}

.Element-right {
  color: white;
  float: right;
  margin-right: 20px;
}
