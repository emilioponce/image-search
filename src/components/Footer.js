import React, { Component } from 'react'

import './Footer.css'

const Footer = () => {
  return (
    <div className="Footer">
      Emilio Ponce.
      <span className="Element">
        <a href="https://www.github.com/emilioponce">Github</a>
      </span>
      <span className="Element">
        <a href="https://emilioponce.info">Blog</a>
      </span>
      <span className="Element-right">
        <a href="https://www.gnu.org/licenses/gpl-3.0.html">GPL-3.0 license</a>
      </span>
    </div>
  )
}

export default Footer
