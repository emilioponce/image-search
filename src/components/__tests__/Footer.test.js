import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Footer from '../Footer'

describe('Footer component', () => {
  const component = <Footer />

  it('Renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(component, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('Snapshot matchs', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
