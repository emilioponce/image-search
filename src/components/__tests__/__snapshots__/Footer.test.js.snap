// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Snapshot matchs 1`] = `
<div
  className="Footer"
>
  Emilio Ponce.
  <span
    className="Element"
  >
    <a
      href="https://www.github.com/emilioponce"
    >
      Github
    </a>
  </span>
  <span
    className="Element"
  >
    <a
      href="https://emilioponce.info"
    >
      Blog
    </a>
  </span>
  <span
    className="Element-right"
  >
    <a
      href="https://www.gnu.org/licenses/gpl-3.0.html"
    >
      GPL-3.0 license
    </a>
  </span>
</div>
`;
